cypress/integration/searchOverlayCompatible/no_results_spec.js

```diff
@@ -32,28 +33,21 @@ describe('Concerning searches with no results:', function () {
     });
   });
 
-  it('When a keyword returns no results, links to other pages are displayed', function () {
-    const noResultsKeyword = 'a7'
+  it('checks links are for the correct environment', () =>{
     search.clearedSearchField.type(noResultsKeyword).then(() => {
-      search.results.noResultsBlock.as('noResultsBlock').should('be.visible');
-
-      //Expected urls exist
-      cy.get('@noResultsBlock').find('[data-automation-id="no-results-corkboard-link"]').as('corkboardLink');
-      cy.get('@corkboardLink').should('be.visible').and('has.attr', 'href').and('contains', `/corkboard`);
-
-      cy.get('@noResultsBlock').find('[data-automation-id="no-results-groups-link"]').as('groupsLink');
-      cy.get('@groupsLink').should('be.visible').and('has.attr', 'href').and('contains', `/groups/search`);
+      cy.get('[data-automation-id="no-results-corkboard-link"]').as('corkboardLink').should('be.visible');
+      cy.get('@corkboardLink').should('have.attr', 'href', `${Cypress.env('CRDS_ENDPOINT')}/corkboard`);
     });
   });
 
-  it('When a successful search is made after a no-results search, results are displayed', function () {
-    const noResultsKeyword = 'a7'
-    search.clearedSearchField.type(noResultsKeyword);
-
-    const resultsKeyword = 'god'
-    search.clearedSearchField.type(resultsKeyword).then(() => {
-      search.results.firstCard.title.should('be.visible');
-      search.results.noResultsBlock.should('not.exist');
+  it('checks results are displayed when a successful search is made after a no-results search', function () {
```

Comment:
@s-henney 

I'm a fan of keeping these short. I'll make a wording suggestion here and you can take it or leave it
```suggestion
  it('checks search results after a no-results search', function () {
```

It's implied you're attempting a valid search because you're saying "after" a no-results search.